site/content/en/docs/tasks/run/multikueue/jobsets.md

```diff
@@ -0,0 +1,4 @@
+The same `jobset-sample.yaml` file from [single cluster environment](docs/tasks/run/jobsets) can be used in a [MultiKueue environment](#multikueue-environment).
```

Comment:
Add intro and "Before you begin" section. It should say that you have MultiKueue setup (referencing the steps).